api/server/models/meetingRequestModel.js

```diff
@@ -1,21 +1,25 @@
-// var mongoose = require("mongoose");
+var mongoose = require("mongoose");
 
-// var Option = require("./optionModel");
+var Option = require("./optionModel");
 
-// var meetingRequestSchema = mongoose.Schema({
-//   author: mongoose.Types.ObjectId,
-//   deadline: { type: Date, required: true },
-//   groupID: mongoose.Types.ObjectId,
-//   multiPoll: [Option]
-//   uniPoll: Option,
-//   status: Number,
-// });
+var meetingRequestSchema = mongoose.Schema({
+  author: {type : mongoose.Types.ObjectId, required: true},
+  deadline: { type: Date, required: true },
+  groupID: {type : mongoose.Types.ObjectId, required: true},
+  multiPoll: [Option],
+  uniPoll: Option,
+  status: Number,
+  isUniPoll: {type : Boolean, required: true},
```

Comment:
Also mentioned above, I think 'Uni' is unclear and unanimous meeting requests shouldn't be considered a 'poll', so I would suggest renaming this to 'isUnanimousRequest' or 'isPollRequest' (or just 'isUnanimous' or 'isPoll').

Reply:
I'll mark this as resolved all because I agree that different variable names are better and I've replied earlier for why I had these names